src/platform/sdl/CMakeLists.txt

```diff
@@ -119,12 +119,13 @@ if(ENABLE_SCRIPTING)
 	endif()
 endif()
 
-add_executable(${BINARY_NAME}-sdl WIN32 ${PLATFORM_SRC} ${MAIN_SRC})
+add_executable(${BINARY_NAME}-sdl ${PLATFORM_SRC} ${MAIN_SRC})
```

Comment:
...Why??

Reply:
Because it's a console app and not a GUI app. `WIN32` means GUI.

Reply:
It's not a console app. It's meant to be launched from frontends and the like so it doesn't have the full GUI.